home/HomeApp.tsx

```diff
@@ -62,43 +60,17 @@ export default function HomeApp() {
   const colorScheme = useColorScheme();
   const preferredAppearance = useSelector((data) => data.settings.preferredAppearance);
   const dispatch = useDispatch();
-  const { accountName, setAccountName } = useAccountName();
+  const { setAccountName } = useAccountName();
```

Comment:
is `setAccountName` used anywhere?

Reply:
yeah it's used when we initialize state and load initial data further down in this file